bundles/net.tourbook.common/src/net/tourbook/common/Messages.java

```diff
@@ -276,15 +276,15 @@
    public static String        Pref_System_Option_Temperature_Celsius;
    public static String        Pref_System_Option_Temperature_Fahrenheit;
 
-   public static String        Weather_Clounds_Clouds;
-   public static String        Weather_Clounds_Clouny;
-   public static String        Weather_Clounds_IsNotDefined;
-   public static String        Weather_Clounds_Lightning;
-   public static String        Weather_Clounds_Rain;
-   public static String        Weather_Clounds_ScatteredShowers;
-   public static String        Weather_Clounds_SevereWeatherAlert;
-   public static String        Weather_Clounds_Snow;
-   public static String        Weather_Clounds_Sunny;
+   public static String        Weather_Clouds_Clouds;
+   public static String        Weather_Clouds_Cloudy;
+   public static String        Weather_Clouds_IsNotDefined;
+   public static String        Weather_Clouds_Lightning;
+   public static String        Weather_Clouds_Rain;
+   public static String        Weather_Clouds_ScatteredShowers;
+   public static String        Weather_Clouds_SevereWeatherAlert;
+   public static String        Weather_Clouds_Snow;
+   public static String        Weather_Clouds_Sunny;
```

Comment:
@wolfgang-ch 
Today i found and fixed those typo issues

Reply:
A typo since 2010 fee487aba0a19dab8cbf29901bee8b114a896086

Reply:
Wow, impressive the nobody as found the typo before me